.github/workflows/deploy-to-stream-dist.yml

```diff
@@ -0,0 +1,129 @@
+name: Deploy to xwp/stream-dist Repository
+
+on:
+  release:
+    types:
+      - released
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question**
Will this run on releases only now? What do you think about keeping `master` and maybe `develop` up to date on the dist repo and Packagist? I know it's under discussion.

Reply:
@tharsheblows, my thought process here was to consider a scenario where Packagist equals wp.org. Since the release branches are based on `develop`, which in turn is based on `master`, we should be covered. I reviewed the Release Cycle to reach this conclusion: https://github.com/xwp/stream/blob/develop/contributing.md#release-cycle. Let me know your thoughts.